tools/vscode/tasks.json

```diff
@@ -0,0 +1,334 @@
+{
+  // Note!
+  // Set the value used for ${config:chrome.outputDir} in your settings.json
+  // file with a line like:
+  //   "chrome.outputDir": "/path/to/chromium/src/out/current_link",
+  // Then run "0-set_chrome_output_directory" to set the `current_link`
+  // symbolic link (see below).
+  "version": "2.0.0",
+  "runner": "terminal",
+  // The default problem matcher matches build output, which is useful for most tasks.
+  "problemMatcher": [
+    {
+      "owner": "cpp",
+      "fileLocation": [
+        "relative",
+        "${config:chrome.outputDir}"
+      ],
+      "pattern": {
+        "regexp": "^(gen/.*):(\\d+):(\\d+):\\s+(warning|\\w*\\s?error):\\s+(.*)$",
+        "file": 1,
+        "line": 2,
+        "column": 3,
+        "severity": 4,
+        "message": 5
+      }
+    },
+    {
+      "owner": "cpp",
+      "fileLocation": [
+        "relative",
+        "${workspaceRoot}"
+      ],
+      "pattern": {
+        "regexp": "^../../(.*):(\\d+):(\\d+):\\s+(warning|\\w*\\s?error):\\s+(.*)$",
+        "file": 1,
+        "line": 2,
+        "column": 3,
+        "severity": 4,
+        "message": 5
+      }
+    },
+    {
+      "owner": "cpp",
+      "fileLocation": [
+        "relative",
+        "${config:chrome.outputDir}"
+      ],
+      "pattern": {
+        "regexp": "^(gen/.*?):(.*):\\s+(warning|\\w*\\s?error):\\s+(.*)$",
+        "file": 1,
+        "severity": 3,
+        "message": 4
+      }
+    },
+    {
+      "owner": "cpp",
+      "fileLocation": [
+        "relative",
+        "${workspaceRoot}"
+      ],
+      "pattern": {
+        "regexp": "^../../(.*?):(.*):\\s+(warning|\\w*\\s?error):\\s+(.*)$",
+        "file": 1,
+        "severity": 3,
+        "message": 4
+      }
+    }
+  ],
+  "options": {
+    "cwd": "${workspaceFolder}/brave",
+    "env": {
+      "PATH": "${workspaceFolder}/third_party/depot_tools:${env:PATH}"
```

Comment:
1. this won't work on windows, because it uses `;` as a path separator. can be solved with:
```
"windows" : {
  "options": {
    "env": {
      "PATH": "${workspaceFolder}/third_party/depot_tools;${env:PATH}"
    }
  }
}
```

2. `${workspaceFolder}/brave/vendor/depot_tools` might be a better choice.

Reply:
I found `${pathSeparator}` in vscode docs for file paths, but I don't see anything for platform independent env paths. I think similar to https://github.com/brave/brave-core/pull/14873/files#r958509120 on windows people will have to make adjustments to it

Reply:
> on windows people will have to make adjustments to it

you can make it work right out of the box. see my previous comment.

what about `${workspaceFolder}/brave/vendor/depot_tools`?

Reply:
I can switch it to brave/vendor, but if I change `:` to `;` then it will break on macos/linux, no?

Reply:
> but if I change : to ; then it will break on macos/linux, no?

it should look like this:
```
  ...
  ],
  "options": {
    "cwd": "${workspaceFolder}/brave",
    "env": {
      "PATH": "${workspaceFolder}/brave/vendor/depot_tools:${env:PATH}"
    }
  },
  "windows": {
    "options": {
      "env": {
        "PATH": "${workspaceFolder}/brave/vendor/depot_tools;${env:PATH}"
      }
    }
  },
  "inputs": [
  ...
```